### DISCUSSION TITLE
No emails send to organizers

### DISCUSSION THREAD

**Discussion by holgerm** [pretix/pretix#3293] [Q&A]
I've activated notifications, I am organizer of an event in test mode, but when a customer buys a ticket, the customre gets an email notification, but the organizer does not get any email. 

Do you have any guess what I'm doing wrong?

Thanks a lot!

Holger

**Comment by Konafets** [comment]
@raphaelm You closed this question as duplicate without providing a link to the duplicate. Can you point me to that? I am facing the same issue.

**Reply by raphaelm** [reply]
This was the duplicate: https://github.com/pretix/pretix/issues/3295
